src/components/CallView/BottomBar.vue

```diff
@@ -310,10 +374,77 @@ useHotKey('r', toggleHandRaised)
 				</template>
 			</NcButton>
 		</div>
-
-		<CallButton
-			:hide-text="isSidebar"
-			:is-screensharing="!!localMediaModel.attributes.localScreen" />
+		<div class="bottom-bar-options call-options">
```

Comment:
instead of `ref="callButton"`, then you cover actions menu as well

```suggestion
		<div ref="callButtonWithActions" class="bottom-bar-options call-options">
```